### PR TITLE
Fix: commands on windowss

### PR DESCRIPTION
**Pull request by narehart**
- fix lint
- fix commit
- fix ts
- fix verify
- fix build

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1485,45 +1485,12 @@
...
-    lodash.clonedeep "4.5.0"
-    lodash.isequal "4.5.0"
-    lodash.template "4.5.0"
-    lodash.uniq "4.5.0"
-    micromatch "4.0.2"
-    p-map "4.0.0"
-    parse-diff "0.7.1"
-    yargs "16.2.0"
-    yarn-deduplicate "1.1.1"
+  version "0.0.0"
```
Comment (ze-flo) [inline yarn.lock:1492]: Why the diff?

File: pkgs/cli-utils/src/fs.ts [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import * as fs from 'fs'
 import * as mm from 'micromatch'
 import fsReadDirRecursive = require('fs-readdir-recursive')
-import { dirname, isAbsolute, resolve } from 'path'
+import { dirname, isAbsolute, posix, resolve, sep } from 'path'
+
+// ensures paths used with mm work cross platform
```
Comment (ze-flo) [inline pkgs/cli-utils/src/fs.ts:6]: 🙄

File: yarn.lock (hunk 1 above)
Comment (narehart) [reply yarn.lock:1492]: Dependabot

Comment (ze-flo) [reply yarn.lock:1492]: dependencies intro'd by dependabot [update](https://github.com/percolate/blend/commit/47f61e1968fb2c031de47896b47c7306495020df). Not clear how. 

Original PR adding `"@percolate/kona@link:./pkgs/kona":` is here: https://github.com/percolate/blend/pull/38/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR1797

Marking as resolved.

File: pkgs/cli-utils/src/fs.ts (hunk 2 above)
Comment (narehart) [reply pkgs/cli-utils/src/fs.ts:6]: path !== glob

**Review by ze-flo — APPROVED** [review]
Awesome work sensei! 💯

**Review by ze-flo — APPROVED** [review]

**Pull request merged**